imagersite/imagersite/tests.py

```diff
@@ -44,3 +44,40 @@ def test_register_user(self):
         link = urlparse(link)
         self.client.get(link.path)
         self.assertTrue(self.client.login(username='person', password='codefellows1'))
+
+
+class ImagesRoutesTests(TestCase):
+    """Class for testing views."""
```

Comment:
These tests belong in the `imager_images` test file because they are testing routes defined in that app.